firstProject/assets/scss/style.scss

```
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
+
+$dark-cyan:hsl(185, 75%, 39%);
+$very-dark-desaturated-blue:hsl(229, 23%, 23%);
+$dark-grayish-blue:hsl(227, 10%, 46%);
+$dark-gray: hsl(0, 0%, 59%);
+
+
+*{
```

```suggestion
* {
```
Kod nije lepo formatiran. Mozes instalirati prettier i on ce to automatski popraviti